frontend/src/router/index.ts

```diff
@@ -176,11 +178,15 @@ router.beforeEach((to, from, next) => {
 
 // THIRD check user/browser keys (requires auth)
 router.beforeEach(async (to) => {
+  let me = undefined;
+  if (!to.meta.skipAuth) {
+    me = await userdata.me;
+    i18n.global.locale.value = mapToLocale(me.language);
+  }
```

Comment:
please move this to a separate `beforeEach` section, just for clarity

Reply:
(`userdata.me` is memoized, so there should be no performance penalty)

Reply:
Fixed in [4b749fb](https://github.com/cryptomator/hub/pull/313/commits/4b749fb53cad8f8fd994b01b9d9ed690d4e5ad33)